configs/raspberrypi4-64-rauc_defconfig

```diff
@@ -58,3 +58,13 @@ BR2_PACKAGE_HOST_UBOOT_TOOLS_ENVIMAGE_SIZE="0x8000"
 BR2_PACKAGE_HOST_UBOOT_TOOLS_ENVIMAGE_REDUNDANT=y
 BR2_PACKAGE_HOST_UBOOT_TOOLS_BOOT_SCRIPT=y
 BR2_PACKAGE_HOST_UBOOT_TOOLS_BOOT_SCRIPT_SOURCE="$(BR2_EXTERNAL_BR2RAUC_PATH)/board/raspberrypi4-64/u-boot_4_64bit.ush"
+# Docker
+BR2_PACKAGE_DROPBEAR=y
+BR2_PACKAGE_DOCKER_CLI=y
+BR2_PACKAGE_DOCKER_CLI_STATIC=y
+BR2_PACKAGE_DOCKER_COMPOSE=y
```

Comment:
Do we still need docker compose? Maybe not, this is something that will no longer be supported.

Reply:
Done